From: Russell King <rmk+kernel@arm.linux.org.uk>
Date: Tue, 27 Mar 2012 10:44:23 +0100
Subject: [PATCH] ARM: fix Kconfig warning for HAVE_BPF_JIT

commit fada8dcf2d085f4e2eb1ba760c8d37111977dbec upstream.

Last night's randconfig and the allnoconfig builds spat out the
following warning while building:

warning: (ARM) selects HAVE_BPF_JIT which has unmet direct dependencies (NET)

Acked-by: Mircea Gherzan <mgherzan@gmail.com>
Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 arch/arm/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux/arch/arm/Kconfig
===================================================================
--- linux.orig/arch/arm/Kconfig	2012-06-24 23:38:52.000000000 +0200
+++ linux/arch/arm/Kconfig	2012-06-24 23:41:24.000000000 +0200
@@ -30,7 +30,7 @@
 	select HAVE_SPARSE_IRQ
 	select GENERIC_IRQ_SHOW
 	select CPU_PM if (SUSPEND || CPU_IDLE)
-	select HAVE_BPF_JIT
+	select HAVE_BPF_JIT if NET
 	help
 	  The ARM series is a line of low-power-consumption RISC chip designs
 	  licensed by ARM Ltd and targeted at embedded applications and
